src/main/java/com/google/sps/utils/data/Animal.java

```diff
@@ -0,0 +1,205 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Collection;
+import java.util.HashSet;
+
+public class Animal {
+    
+    // Cross-referenced
+    private String common_name;
+    private String binomial_name;
+
+    // Provided by Wikipedia
+    private int population;
+    private String status;
+
+    // Provided by IUCN API
+    private int id;
+    private String kingdom;
```

Comment:
Do you think it'd be a good idea to create a Taxonomy class with all of these variables to group them together?

Reply:
Yeah sure!